frontend/packages/ux-editor/src/components/Settings/SettingsUtils.tsx

```diff
@@ -0,0 +1,37 @@
+import React, { type ReactElement } from 'react';
+import {
+  NavigationDataIcon,
+  QuestionmarkIcon,
+  NavigationSignIcon,
+  NavigationPayIcon,
+  NavigationReceiptIcon,
+} from '@studio/icons';
+import { PROTECTED_TASK_NAME_CUSTOM_RECEIPT } from 'app-shared/constants';
+
+export const taskNavigationType = (taskType: string) => {
+  if (taskType === 'receipt' || taskType === PROTECTED_TASK_NAME_CUSTOM_RECEIPT) {
+    return 'process_editor.configuration_panel_custom_receipt_accordion_header';
+  } else {
+    return `process_editor.task_type.${taskType}`;
+  }
+};
+
+export const taskNavigationIcon = (taskType: string, taskIcon?: string): ReactElement => {
+  switch (taskType) {
+    case 'receipt':
+    case PROTECTED_TASK_NAME_CUSTOM_RECEIPT:
+      return <NavigationReceiptIcon data-testid='receipt' className={taskIcon} />;
+    case 'data':
+      return <NavigationDataIcon data-testid='data' className={taskIcon} />;
+    case 'signing':
+      return <NavigationSignIcon data-testid='signing' className={taskIcon} />;
+    case 'payment':
+      return <NavigationPayIcon data-testid='payment' className={taskIcon} />;
```

Comment:
I think we need to add an extra case here to support "feedback" and "confirmation" types:
```suggestion
      return <NavigationPayIcon data-testid='payment' className={taskIcon} />;
    case 'feedback':
      ...
    case 'confirmation':
      ...
```

![confirm](https://github.com/user-attachments/assets/6c86f49c-b0d3-4206-9d99-b8507d94b92d)